notebooks/240-dolly-2-instruction-following/240-dolly-2-instruction-following.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
```

Comment:
<p>for now (until 2023.2.0 prerelease is not published, we need to add disclamer that weights compression do not have effect on GPU)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added